radix-engine-stores/src/rocks_db.rs

```diff
@@ -80,10 +80,6 @@ impl RadixEngineDB {
 
     pub fn list_components(&self) -> Vec<ComponentAddress> {
         let mut addresses = Vec::new();
-        addresses.extend(self.list_components_helper(
-            ComponentAddress::System([0u8; 26]),
-            ComponentAddress::System([255u8; 26]),
```

Comment:
Let's talk to Russ/Matt about this change as like we talked about I am under the impression that we only want a few types of top level things: Component/Resource/Package.

Reply:
Yes, I will add it to our to discuss list.